### ISSUE TITLE
Add a `relish push` line to continuous integration 

### ISSUE THREAD

**Issue by Stephen-Gates** [ODIQueensland/data-curator#10]
adding a `relish push` to continuous integration would mean that [features published to Relish](https://relishapp.com/odi-australia/data-curator/docs) would always be up to date

```
gem install relish
relish push <path>
```

**Comment by Stephen-Gates** [comment]
may not be needed after #496 tested

**Comment by Stephen-Gates** [comment]
no longer needed - using Cucumber Pro
